chart/templates/mayastor-daemonset.yaml

```diff
@@ -56,6 +56,8 @@ spec:
         - "-P/var/local/mayastor/pools.yaml"
         - "-l{{ include "mayastorCpuSpec" . }}"
         - "-pmayastor-etcd"
+        command:
+        - mayastor
```

Comment:
* nit: this specific change is a separate logical change so best to put it in a separate patch/commit with its own description/rationale. 

* Talking of rationales, the commit log is missing that. Please also provide it on the commits, not just the PR so it can be view quickly and/or offline.

Reply:
I don't think it is a separate logical change, we will be using the same image for mayastor and maystor-csi.
So for consistency both yaml files should have the command argument.

Reply:
I've just updated the git commit message to match the PR description.
The change in the mayastor-daemonset.yaml is related to the change in mayastor-csi-daemonset.yaml in that they both use the same image and so should use the "command" field to specify the binary to run. I have updated the comments to reflect this.

Reply:
> I don't think it is a separate logical change, we will be using the same image for mayastor and maystor-csi.

Separate logical change in the sense of commit, not PR.